pkg/apis/sources/v1alpha1/jobsource_defaults.go

```diff
@@ -25,4 +29,20 @@ func (js *JobSource) SetDefaults(ctx context.Context) {
 	if js.Spec.OutputFormat == "" {
 		js.Spec.OutputFormat = OutputFormatBinary
 	}
+
+	// Use the documented default for the embedded JobSpec.
+	// See k8s.io/api/batch/v1.JobSpec.BackoffLimit.
+	if js.Spec.BackoffLimit == nil {
+		js.Spec.BackoffLimit = ptr.Int32(6)
```

Comment:
can we use `k8s.io/api/batch/v1.JobSpec.BackoffLimit` directly? or is it just a comment?

Reply:
Unfortunately not; this constant only appears in documentation.